Added a 1602 LCD display with I2C interface module

pcremades opened the pull request:
I added a [1/4" square trimpot](https://ar.mouser.com/ProductDetail/Bourns/3362F-1-254LF?qs=%2FxQVPCMPNzjxPTCOgN1%2Fdw%3D%3D) to the libary.

![image](https://github.com/user-attachments/assets/0bc54673-e86a-4036-a7c2-3eb6e0c6d547)

Added a pcb for the 1602 LCD I2C interface

![image](https://github.com/user-attachments/assets/d2bf0f21-ca22-4c5e-ac10-3b402dae3b80)

Finally, I added a new 1602 LCD display with the I2C module incorporated.

![image](https://github.com/user-attachments/assets/de894fa3-21ac-46fe-a0e8-f86677c2e6e5)

nophead commented:
Nice, let me know when you have finished making changes.

pcremades commented:
Hi Chris,
The changes are ready for merging. I don't know how to stop new commits from being added to the same pull request. Because now I'm working on a new potentiometer, but that is not related to this pull request.

nophead commented:
You need to create a new branch and work on that instead of the branch this pull request is on. It looks like you are using your master branch. It is more usual to create a new branch before making changes that destined for a pull request.

pcremades commented:
Ah, ok. I'm used to GitLab, which lets you specify which commits to include in a 'merge request'. I'm switching to a different branch then.

pcremades commented:
Hi Chris,
I moved my most recent commits to a new branch and hard reset master to the point where I added the LCD. So now, the pull request should be ready for merging.

nophead commented:
OK I look at merging it later today or tomorrow.
